app/views/concerns/entity_displaying_view.rb

```diff
@@ -205,7 +205,9 @@ def entity_death
   end
 
   def entity_date(dates)
-    date_most_accurate(dates)
+    return dates if dates.is_a?(String)
```

Comment:
we still need to apply `date_eras_gregorian(dates)` and possibly `.strip` even if passed a string.
rather than returning dates when entity_date is passed a string, either assign dates = [dates] or remove this line altogether and leave it to the calling method to supply an array.